Lib/xml/dom/minidom.py

```diff
@@ -882,7 +898,7 @@ def getElementsByTagNameNS(self, namespaceURI, localName):
             self, namespaceURI, localName, NodeList())
 
     def __repr__(self):
-        return "<DOM Element: %s at %#x>" % (self.tagName, id(self))
+        return f"<DOM Element: {self.tagName}>"
```

Comment:
This breaks the backward compatibility. But should we care about the programs that depend on `__repr__` ?
Isn't that implementation detail?

Reply:
This is off-topic for a docstring patch, which I believe can otherwise be backported.  Please delete.  (In case you don't know, please do not squash commits.  )

Reply:
Addressed in  b0a91293f00